app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb

```diff
@@ -0,0 +1,91 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach English</h2>
+    <div class="inset">
+      <p>As an English teacher, you'll be a cornerstone in pupils' lives. From reading and exploring literature to writing and delivering debates, you'll cover essential skills needed to think critically and thrive beyond the classroom.</p>
```

Comment:
I can't remember if I wrote this or you wrote this, but not sure I understand the first sentence? Feels more like being a cornerstone would be the role of a form tutor maybe?